README.md

```diff
@@ -92,105 +75,103 @@ You can use enry as a command,
+enry, A simple (and faster) implementation of github/linguist
+usage: enry [-mode=(file|line|byte)] [-prog] <path>
+        enry [-mode=(file|line|byte)] [-prog] [-json] [-breakdown] <path>
+        enry [-mode=(file|line|byte)] [-prog] [-json] [-breakdown]
+        enry [-version]
 ```
 
-and it'll return an output similar to *linguist*'s output,
+and on repository root, it'll return an output similar to *linguist*'s output,
 
 ```bash
 $ enry
-55.56%    Shell
-22.22%    Ruby
-11.11%    Gnuplot
-11.11%    Go
+97.71%	Go
+1.60%	C
+0.31%	Shell
+0.22%	Java
+0.07%	Ruby
+0.05%	Makefile
+0.04%	Scala
+0.01%	Gnuplot
 ```
 
 but not only the output; its flags are also the same as *linguist*'s ones,
 
 ```bash
 $ enry --breakdown
-55.56%    Shell
-22.22%    Ruby
-11.11%    Gnuplot
-11.11%    Go
-
-Gnuplot
-plot-histogram.gp
-
-Ruby
-linguist-samples.rb
-linguist-total.rb
-
-Shell
-parse.sh
-plot-histogram.sh
-run-benchmark.sh
-run-slow-benchmark.sh
-run.sh
+97.71%	Go
+1.60%	C
+0.31%	Shell
+0.22%	Java
+0.07%	Ruby
+0.05%	Makefile
+0.04%	Scala
+0.01%	Gnuplot
+
+Scala
+java/build.sbt
+java/project/plugins.sbt
+
+Java
+java/src/main/java/tech/sourced/enry/Enry.java
+java/src/main/java/tech/sourced/enry/GoUtils.java
+java/src/main/java/tech/sourced/enry/Guess.java
+java/src/test/java/tech/sourced/enry/EnryTest.java
+
+Makefile
+Makefile
+java/Makefile
 
 Go
-parser/main.go
+benchmark_test.go
 ```
 
 even the JSON flag,
 
 ```bash
-$ enry --json
-{"Gnuplot":["plot-histogram.gp"],"Go":["parser/main.go"],"Ruby":["linguist-samples.rb","linguist-total.rb"],"Shell":["parse.sh","plot-histogram.sh","run-benchmark.sh","run-slow-benchmark.sh","run.sh"]}
+$ enry --json | jq .
+{
+  "C": [
+    "internal/tokenizer/flex/lex.linguist_yy.c",
+    "internal/tokenizer/flex/lex.linguist_yy.h",
+    "internal/tokenizer/flex/linguist.h",
+    "python/_c_enry.c",
+    "python/enry.c"
+  ],
+  "Gnuplot": [
+    "benchmarks/plot-histogram.gp"
+  ],
+  "Go": [
+    "benchmark_test.go",
 ```
 
-Note that even if enry's CLI is compatible with linguist's, its main point is that **_enry doesn't need a git repository to work!_**
-
-Java bindings
-------------
-
-Generated Java bindings using a C-shared library and JNI are located under [`java`](https://github.com/src-d/enry/blob/master/java)
-
-Development
-------------
-
-*enry* re-uses parts of original [linguist](https://github.com/github/linguist) to generate internal data structures. In order to update to the latest upstream and generate all the necessary code you must run:
-
-    git clone https://github.com/github/linguist.git .linguist
-    # update commit in generator_test.go (to re-generate .gold fixtures)
-    # https://github.com/src-d/enry/blob/13d3d66d37a87f23a013246a1b0678c9ee3d524b/internal/code-generator/generator/generator_test.go#L18
-    go generate
+Note that enry's CLI **_doesn't need a git repository to work_**, which is intentionally different from the linguist.
```

Comment:
```suggestion
Note that enry's CLI **_doesn't need a git repository to work_**, which is intentionally different from linguist.
```

But I guess that this is also correlated with limitations like not using `.gitattributes` files?

Reply:
It's different - linguist is based on using git to get file content, Enry is not.
`.gitatributes` is just something that was never implemented in enry as large-scale parsing use case we care about did not requir it.